index.json

```
@@ -38,6 +38,8 @@
     "requires": [
       "active_support",
       "active_support/core_ext/array",
+      "active_support/core_ext/string/starts_ends_with.rb",
+      "active_support/core_ext/symbol/starts_ends_with.rb",
       "active_support/test_case"
```

Should we just do this instead?

```suggestion
      "active_support/all"
```